src/styled/developer-topics.ts

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled';
+
+const StyledButton = styled('button')`
+    border: none;
+    background: none;
+    color: white;
```

Comment:
defining colors as `hex` values are faster than defining them as named colors. So `#fff `would be better